dashboard/urls.py

```diff
@@ -33,8 +33,9 @@
 urlpatterns = [
     path('', views.get_home_template, name = 'home'),
     path('status/', include('watchman.urls')),
-    path('status/bare_status', watchman.views.bare_status),
-    path('admin', admin.site.urls),
+    path('status/bare_status/', watchman.views.bare_status),
+
+    path('admin/', admin.site.urls),
 
     # graphql access
     path('graphql', DashboardGraphQLView.as_view(graphiql=True)),
```

Comment:
I tried adding a `/` here but that seemed to break the GraphQL implementation. Part of the problem seems to be that `PUT` calls will lose data when the slash is appended and a redirect happens. I was hoping there would be a way to add `/` to calls in the front-end -- like we did with `userSetUserSetting` -- but I wasn't able to find anything easily. @justin0022, any suggestions? I'm fine keeping it without the `/` though, it'd be nice to be consistent.

Reply:
Hey sorry @ssciolla I'll look into this. Didn't see this message till now.

Reply:
I'm not sure what the best way to address this is

Reply:
K, let's leave it alone for now.